docs/installation.md

```diff
@@ -15,13 +14,51 @@ We currently recommend installing the development version of owlkettle.
 $ nimble install owlkettle@#head
 ```
 
+## Properly setting up MSYS2 on Windows
+
+For the Windows OS you'll need the MSYS2 
+that you can install from their [official website](https://www.msys2.org/).
+
+Then, you'll need to use a toolchain with essential binaries as compilers, linkers 
+and the `pkg-config` as well.
+
+There are 2 noteworthy toolchain provided by the MSYS2:
+
+A) `CLANG64`: the recommended one for Nim:
```

Comment:
Superfluous colon at the end ;-)

Also with the current way of formulating this I get the mild impression that you might be recommending installing both. Afaik you should only install one (?).
Thus it would make sense to rephrase line 25 to make that clearer, like appending "Install one of them".

Rest LGTM!

Reply:
Alright!

Reply:
Done!